java/java.lsp.server/nbcode/integration/src/org/netbeans/modules/nbcode/integration/layer.xml

```diff
@@ -21,10 +21,26 @@
 -->
 <!DOCTYPE filesystem PUBLIC "-//NetBeans//DTD Filesystem 1.2//EN" "http://www.netbeans.org/dtds/filesystem-1_2.dtd">
 <filesystem>
+    <folder name="Services">
+        <folder name="NBLS">
+            <file name="general-node-commands.instance">
+                <attr name="instanceCreate" methodvalue="org.netbeans.modules.java.lsp.server.explorer.NodeActionsProvider.forFile"/>
+                <attr name="action:org.openide.actions.DeleteAction" stringvalue="Edit"/>
+            </file>
+            <file name="database-connections-commands.instance">
```

Comment:
Is this registration necessary? Can't all actions registered under `Actions/` folder be available?

Reply:
hm ! What a thought! They could, although I'd prefer to explicitly enable just those actions that were reviewed to use Swing UI wisely (= use UI abstractions where possible). Most of actions are really prepared to work from the server mode.